int32 decision_state
float32 angular_error
float32 correction